package.json

```diff
@@ -1,15 +1,23 @@
 {
-  "name": "book-store-react",
+  "name": "bookstore-project",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
-    "web-vitals": "^2.1.4"
+    "@testing-library/jest-dom": "^5.14.1",
```

Comment:
- [ ] Kindly add `redux toolkit` to your project to achieve this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=Add%20Redux%20Toolkit(npm%20install%20react%2Dredux%20%40reduxjs/toolkit).) .